### PR TITLE
VZ-9538: Build VAO, VPO and Cluster Operator images using Verrazzano base image

### PR DESCRIPTION
**Pull request by bhatpmk**
This change builds images for platform operator, application operator and cluster operator, using the latest Verrazzano base image.

### CODE REVIEW

File: cluster-operator/Dockerfile [hunk 1]
```diff
@@ -13,7 +14,7 @@ RUN chmod 500 /usr/local/bin/verrazzano-cluster-operator \
     && chmod 500 /root/go/src/github.com/verrazzano/cluster-operator/scripts/wait4webhook.sh
 
 # Create the verrazzano-cluster-operator image
-FROM ghcr.io/oracle/oraclelinux:8-slim AS final
+FROM $BASE_IMAGE AS final
 
 RUN microdnf update -y \
```
Comment (kminder) [inline cluster-operator/Dockerfile:19]: Doesn't doing a microdnf update -y ruin all the benefits of using the base image?

File: application-operator/Dockerfile [hunk 2]
```diff
@@ -13,7 +14,7 @@ RUN chmod 500 /usr/local/bin/verrazzano-application-operator \
     && chmod 500 /root/go/src/github.com/verrazzano/application-operator/scripts/wait4webhook.sh
 
 # Create the verrazzano-application-operator image
-FROM ghcr.io/oracle/oraclelinux:8-slim AS final
+FROM $BASE_IMAGE AS final
 
 RUN microdnf update -y \
```
[the remark this answers is not in the document]
Comment (bhatpmk) [reply application-operator/Dockerfile]: Updated

File: cluster-operator/Dockerfile (hunk 1 above)
Comment (bhatpmk) [reply cluster-operator/Dockerfile]: Updated the Dockerfile

**Review by kminder — APPROVED** [review]
Approved.

**Comment by bhatpmk** [comment]
VZ-9538

**Pull request merged**